r/build.gradle

```diff
@@ -73,12 +73,20 @@ task test(type: Exec, dependsOn: distR) {
 
 task installH2ORPackage(type: Exec) {
     def majorName = (h2oMajorName == "master") ? "master" : "rel-${h2oMajorName}"
-    commandLine getOsSpecificCommandLine(["R", "-e", "install.packages(\"h2o\", type = \"source\", repos = \"http://h2o-release.s3.amazonaws.com/h2o/${majorName}/${h2oBuild}/R\")"])
+    commandLine getOsSpecificCommandLine([
+        "R",
+        "-e",
+        "install.packages(\"h2o\", type = \"source\", repos = \"http://h2o-release.s3.amazonaws.com/h2o/${majorName}/${h2oBuild}/R\")"
+    ])
 }
 
 task installRSparklingPackage(type: Exec, dependsOn: distR) {
     workingDir project.buildDir
-    commandLine getOsSpecificCommandLine(["R", "-e", "install.packages(\"rsparkling_${version}.tar.gz\", type=\"source\", repos=NULL)"])
+    commandLine getOsSpecificCommandLine([
```

Comment:
It would be more legible if it was on one line

Reply:
Yup, fixed